AUTHORS

```diff
@@ -8,5 +8,6 @@
 #
 # Please keep the list sorted.
 
+Antonio Marcedone <a.marcedone@gmail.com>
```

Comment:
OK. What got me is that, apparently, we're missing a bunch of people at AUTHORS, so I understood their absence as a sign that we shouldn't do that.